### DISCUSSION TITLE
I can't tell how Keka works

### DISCUSSION THREAD

**Discussion by NBaLogn** [aonez/Keka#1519] [Q&A] [answered]
As the title says, I was just wondering which commands Keka use when executing compression when I choose the Store or Slow method. Take zstd for example, you can adjust compression levels, threads, memory, logs, etc. 

So I was snooping through the code, and I just couldn't figure out where to find it. I downloaded the source file for the latest version, decompressed all the files in the bin folder, but they seemed like all standard libraries. I checked the commits, but it's all related to translations. 

Can sb point me to it?

**Comment by aonez** [comment, marked as answer]
The Keka source has been closed for some years now.

You can create a debug session and check the logs to see the commands used with any format and configuration:

https://github.com/aonez/Keka/wiki/Report-an-issue#debug-session
